### ISSUE TITLE
[FEATURE]: Give the possibility to the warden to change their C4 username

### ISSUE THREAD

**Issue by Simon-Busch** [code-423n4/code423n4.com#2358]
## Problem / opportunity:
Wardens may want to be able to change their C4 username.

## Desired outcome
Once ready, they will be able to change their warden name

## Solution
Create a function to update it in the account management page.
It will also have a big of impact:

- [ ] Updating their username in their previous findings
- [ ] Updating their username in previous reports
- [ ] ....

**Comment by Simon-Busch** [comment]
This has been discussed and agreed with @captainmangoC4 right before warden auth launch

**Comment by sockdrawermoney** [comment]
The impact is significantly deeper than this until we have a method for tracking historical data.

**Comment by sockdrawermoney** [comment]
Specifically:

We are about to run a mason project which will pull in data from past repos. This will depend on username consistency.

The script would also have to update the json files (and ideally commit messages) for all past findings repos, and we'd want to know that it didn't break the past data.

I don't actually think this is a practical thing to do and instead we should wait until we have:

1. completely analyzed past data,
2. created a completely data-abstracted way to universally swap out a uuids for handles, and
3. added tests built on the data analysis in (1) to ensure that changes made in (2) reconcile for the sake of consistency.

**Comment by Simon-Busch** [comment]
I didn't know about the mason project, but it seems great ! 

The idea in opening this issue was that, as we removed this functionality from warden-auth, we just keep track of this. 

This morning we started discussing about it with @captainmangoC4 and both realized that the impact of such a change is quite deep and therefore decided to put it on hold before we can surely scope the real impact of it. 

> created a completely data-abstracted way to universally swap out all uuids for handles, and 

I remember you asked about uuid in Moralis. 

Should this be how wardens are primarily referenced in the code disregarding their handles, discord username etc. ... ?
